pkg/api/handlers/compat/containers_stop.go

```diff
@@ -23,9 +23,10 @@ func StopContainer(w http.ResponseWriter, r *http.Request) {
 
 	// /{version}/containers/(name)/stop
 	query := struct {
-		Ignore        bool `schema:"ignore"`
-		DockerTimeout uint `schema:"t"`
-		LibpodTimeout uint `schema:"timeout"`
+		Ignore        bool                `schema:"ignore"`
+		DockerTimeout uint                `schema:"t"`
+		LibpodTimeout uint                `schema:"timeout"`
+		Filters        map[string][]string `schema:"filters"`
```

Comment:
I'll just add my usual whine that having the variables in a struct listed alphabetically is preferred, at least by my OCD.